core/src/idx/trees/btree.rs

```diff
@@ -33,6 +33,9 @@ pub struct BState {
 	minimum_degree: u32,
 	root: Option<NodeId>,
 	next_node_id: NodeId,
+	#[serde[skip]]
+	// Not used anymore
+	_updated: bool,
 	#[revision(start = 2)]
```

Comment:
This fixes the issue of migrating from 1.0.x and 1.1.x.
But then it breaks the migration from 1.2.0 and 1.2.1

Reply:
This is fixed now, and migrate properly from any older version